src/wmain/qserv-worker-http.cc

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
Why make another module for one class? Why not to put this class and `qserv-worker-http` into an existing module `worker`?

Reply:
The things in `worker`  now have very little to do with anything in `wmain` or `wcomms`. I'm trying to look at this from the point of view of me trying to find something after 5 years have gone by without touching the code, or someone new. I'm still thinking over names and locations.